### ISSUE TITLE
[LocalNet] Errors Starting LocalNet

### ISSUE THREAD

**Issue by 0xThresh** [pokt-network/poktroll#445]
## Issue
While attempting to follow the [LocalNet](https://dev.poktroll.com/infrastructure/localnet) documentation, I am running into issues with `make localnet_up` that prevent me from running the local network. I have double-checked that all dependencies are installed, versions match the requirements outlined in the Makefile, and that I can manually run commands that are listed in the Makefile functions. The full log output is in the Logs section below.

## Tool Versions
Golang: v1.22.0
Ignite: v28.3.0
Kind: v0.22.0
NPM: v8.19.4
NodeJS: v16.20.0
Docker: v24.0.5
Helm: v3.12.2


## Logs
```
make localnet_up
✔ Generated Go code
Processing module application
Modifying file: ./api/poktroll/application/genesis.pulsar.go
Modifying file: ./api/poktroll/application/query.pulsar.go
Processing module gateway
Processing module proof
Processing module service
Processing module session
Modifying file: ./api/poktroll/session/query.pulsar.go
Processing module shared
Processing module supplier
Processing module tokenomics
+----------------------------------------------------------------------------------+
|                                                                                  |
|     IMPORTANT: Please run the following command once to initialize E2E tests     |
|     after the network has started:                                               |
|         make acc_initialize_pubkeys                                              |
|                                                                                  |
+----------------------------------------------------------------------------------+
Initializing chain...
Cosmos SDK's version is: v0.50.4

✘ Cannot build app:                                                                                                                                                                                                                                                                                                       
                                                                                                                                                                                                                                                                                                                          
error while running command go build -o /Users/thresh/go/bin/poktrolld -mod readonly -tags  -ldflags -X github.com/cosmos/cosmos-sdk/version.Name=Poktroll -X github.com/cosmos/cosmos-sdk/version.AppName=poktrolld -X github.com/cosmos/cosmos-sdk/version.Version= -X github.com/cosmos/cosmos-sdk/version.Commit=30e796b5b9cec6cee160b300c28ef13afe254bd7 -X github.com/cosmos/cosmos-sdk/version.BuildTags= -X github.com/pokt-network/poktroll/cmd/poktrolld/cmd.ChainID=poktroll .: # github.com/pokt-network/poktroll/x/shared/types
../../x/shared/types/service.pb 3.go:29:6: RPCType redeclared in this block                                                                                                                                                                                                                                               
	../../x/shared/types/service.pb 2.go:29:6: other declaration of RPCType                                                                                                                                                                                                                                           
../../x/shared/types/service.pb 3.go:32:2: RPCType_UNKNOWN_RPC redeclared in this block                                                                                                                                                                                                                                   
	../../x/shared/types/service.pb 2.go:32:2: other declaration of RPCType_UNKNOWN_RPC                                                                                                                                                                                                                               
../../x/shared/types/service.pb 3.go:33:2: RPCType_GRPC redeclared in this block                                                                                                                                                                                                                                          
	../../x/shared/types/service.pb 2.go:33:2: other declaration of RPCType_GRPC                                                                                                                                                                                                                                      
../../x/shared/types/service.pb 3.go:34:2: RPCType_WEBSOCKET redeclared in this block                                                                                                                                                                                                                                     
	../../x/shared/types/service.pb 2.go:34:2: other declaration of RPCType_WEBSOCKET                                                                                                                                                                                                                                 
../../x/shared/types/service.pb 3.go:35:2: RPCType_JSON_RPC redeclared in this block                                                                                                                                                                                                                                      
	../../x/shared/types/service.pb 2.go:35:2: other declaration of RPCType_JSON_RPC                                                                                                                                                                                                                                  
../../x/shared/types/service.pb 3.go:36:2: RPCType_REST redeclared in this block                                                                                                                                                                                                                                          
	../../x/shared/types/service.pb 2.go:36:2: other declaration of RPCType_REST                                                                                                                                                                                                                                      
../../x/shared/types/service.pb 3.go:39:5: RPCType_name redeclared in this block                                                                                                                                                                                                                                          
	../../x/shared/types/service.pb 2.go:39:5: other declaration of RPCType_name                                                                                                                                                                                                                                      
../../x/shared/types/service.pb 3.go:47:5: RPCType_value redeclared in this block                                                                                                                                                                                                                                         
	../../x/shared/types/service.pb 2.go:47:5: other declaration of RPCType_value                                                                                                                                                                                                                                     
../../x/shared/types/service.pb 3.go:65:6: ConfigOptions redeclared in this block                                                                                                                                                                                                                                         
	../../x/shared/types/service.pb 2.go:65:6: other declaration of ConfigOptions                                                                                                                                                                                                                                     
../../x/shared/types/service.pb 3.go:68:2: ConfigOptions_UNKNOWN_CONFIG redeclared in this block                                                                                                                                                                                                                          
	../../x/shared/types/service.pb 2.go:68:2: other declaration of ConfigOptions_UNKNOWN_CONFIG                                                                                                                                                                                                                      
../../x/shared/types/service.pb 3.go:68:2: too many errors                                                                                                                                                                                                                                                                
# github.com/pokt-network/poktroll/api/poktroll/application                                                                                                                                                                                                                                                               
../../api/poktroll/application/query_grpc.pb 3.go:22:2: Query_Params_FullMethodName redeclared in this block                                                                                                                                                                                                              
	../../api/poktroll/application/query_grpc.pb 2.go:22:2: other declaration of Query_Params_FullMethodName                                                                                                                                                                                                          
../../api/poktroll/application/query_grpc.pb 3.go:23:2: Query_Application_FullMethodName redeclared in this block                                                                                                                                                                                                         
	../../api/poktroll/application/query_grpc.pb 2.go:23:2: other declaration of Query_Application_FullMethodName                                                                                                                                                                                                     
../../api/poktroll/application/query_grpc.pb 3.go:24:2: Query_AllApplications_FullMethodName redeclared in this block                                                                                                                                                                                                     
	../../api/poktroll/application/query_grpc.pb 2.go:24:2: other declaration of Query_AllApplications_FullMethodName                                                                                                                                                                                                 
../../api/poktroll/application/query_grpc.pb 3.go:30:6: QueryClient redeclared in this block                                                                                                                                                                                                                              
	../../api/poktroll/application/query_grpc.pb 2.go:30:6: other declaration of QueryClient                                                                                                                                                                                                                          
../../api/poktroll/application/query_grpc.pb 3.go:38:6: queryClient redeclared in this block                                                                                                                                                                                                                              
	../../api/poktroll/application/query_grpc.pb 2.go:38:6: other declaration of queryClient                                                                                                                                                                                                                          
../../api/poktroll/application/query_grpc.pb 3.go:42:6: NewQueryClient redeclared in this block                                                                                                                                                                                                                           
	../../api/poktroll/application/query_grpc.pb 2.go:42:6: other declaration of NewQueryClient                                                                                                                                                                                                                       
../../api/poktroll/application/query_grpc.pb 3.go:76:6: QueryServer redeclared in this block                                                                                                                                                                                                                              
	../../api/poktroll/application/query_grpc.pb 2.go:76:6: other declaration of QueryServer                                                                                                                                                                                                                          
../../api/poktroll/application/query_grpc.pb 3.go:86:6: UnimplementedQueryServer redeclared in this block                                                                                                                                                                                                                 
	../../api/poktroll/application/query_grpc.pb 2.go:86:6: other declaration of UnimplementedQueryServer                                                                                                                                                                                                             
../../api/poktroll/application/query_grpc.pb 3.go:103:6: UnsafeQueryServer redeclared in this block                                                                                                                                                                                                                       
	../../api/poktroll/application/query_grpc.pb 2.go:103:6: other declaration of UnsafeQueryServer                                                                                                                                                                                                                   
../../api/poktroll/application/query_grpc.pb 3.go:107:6: RegisterQueryServer redeclared in this block                                                                                                                                                                                                                     
	../../api/poktroll/application/query_grpc.pb 2.go:107:6: other declaration of RegisterQueryServer                                                                                                                                                                                                                 
../../api/poktroll/application/query_grpc.pb 3.go:107:6: too many errors                                                                                                                                                                                                                                                  
# github.com/pokt-network/poktroll/x/gateway/types                                                                                                                                                                                                                                                                        
../../x/gateway/types/gateway.pb 3.go:27:6: Gateway redeclared in this block                                                                                                                                                                                                                                              
	../../x/gateway/types/gateway.pb 2.go:27:6: other declaration of Gateway                                                                                                                                                                                                                                          
../../x/gateway/types/gateway.pb 3.go:63:5: xxx_messageInfo_Gateway redeclared in this block                                                                                                                                                                                                                              
	../../x/gateway/types/gateway.pb 2.go:63:5: other declaration of xxx_messageInfo_Gateway                                                                                                                                                                                                                          
../../x/gateway/types/gateway.pb 3.go:85:5: fileDescriptor_3b77f5ee1eae982b redeclared in this block                                                                                                                                                                                                                      
	../../x/gateway/types/gateway.pb 2.go:85:5: other declaration of fileDescriptor_3b77f5ee1eae982b                                                                                                                                                                                                                  
../../x/gateway/types/gateway.pb 3.go:147:6: encodeVarintGateway redeclared in this block                                                                                                                                                                                                                                 
	../../x/gateway/types/gateway.pb 2.go:147:6: other declaration of encodeVarintGateway                                                                                                                                                                                                                             
../../x/gateway/types/gateway.pb 3.go:175:6: sovGateway redeclared in this block                                                                                                                                                                                                                                          
	../../x/gateway/types/gateway.pb 2.go:175:6: other declaration of sovGateway                                                                                                                                                                                                                                      
../../x/gateway/types/gateway.pb 3.go:178:6: sozGateway redeclared in this block                                                                                                                                                                                                                                          
	../../x/gateway/types/gateway.pb 2.go:178:6: other declaration of sozGateway                                                                                                                                                                                                                                      
../../x/gateway/types/gateway.pb 3.go:299:6: skipGateway redeclared in this block                                                                                                                                                                                                                                         
	../../x/gateway/types/gateway.pb 2.go:299:6: other declaration of skipGateway                                                                                                                                                                                                                                     
../../x/gateway/types/gateway.pb 3.go:379:2: ErrInvalidLengthGateway redeclared in this block                                                                                                                                                                                                                             
	../../x/gateway/types/gateway.pb 2.go:379:2: other declaration of ErrInvalidLengthGateway                                                                                                                                                                                                                         
../../x/gateway/types/gateway.pb 3.go:380:2: ErrIntOverflowGateway redeclared in this block                                                                                                                                                                                                                               
	../../x/gateway/types/gateway.pb 2.go:380:2: other declaration of ErrIntOverflowGateway                                                                                                                                                                                                                           
../../x/gateway/types/gateway.pb 3.go:381:2: ErrUnexpectedEndOfGroupGateway redeclared in this block                                                                                                                                                                                                                      
	../../x/gateway/types/gateway.pb 2.go:381:2: other declaration of ErrUnexpectedEndOfGroupGateway                                                                                                                                                                                                                  
../../x/gateway/types/gateway.pb 3.go:381:2: too many errors                                                                                                                                                                                                                                                              
# github.com/pokt-network/poktroll/api/poktroll/gateway                                                                                                                                                                                                                                                                   
../../api/poktroll/gateway/query_grpc.pb 3.go:22:2: Query_Params_FullMethodName redeclared in this block                                                                                                                                                                                                                  
	../../api/poktroll/gateway/query_grpc.pb 2.go:22:2: other declaration of Query_Params_FullMethodName                                                                                                                                                                                                              
../../api/poktroll/gateway/query_grpc.pb 3.go:23:2: Query_Gateway_FullMethodName redeclared in this block                                                                                                                                                                                                                 
	../../api/poktroll/gateway/query_grpc.pb 2.go:23:2: other declaration of Query_Gateway_FullMethodName                                                                                                                                                                                                             
../../api/poktroll/gateway/query_grpc.pb 3.go:24:2: Query_AllGateways_FullMethodName redeclared in this block                                                                                                                                                                                                             
	../../api/poktroll/gateway/query_grpc.pb 2.go:24:2: other declaration of Query_AllGateways_FullMethodName                                                                                                                                                                                                         
../../api/poktroll/gateway/query_grpc.pb 3.go:30:6: QueryClient redeclared in this block                                                                                                                                                                                                                                  
	../../api/poktroll/gateway/query_grpc.pb 2.go:30:6: other declaration of QueryClient                                                                                                                                                                                                                              
../../api/poktroll/gateway/query_grpc.pb 3.go:38:6: queryClient redeclared in this block                                                                                                                                                                                                                                  
	../../api/poktroll/gateway/query_grpc.pb 2.go:38:6: other declaration of queryClient                                                                                                                                                                                                                              
../../api/poktroll/gateway/query_grpc.pb 3.go:42:6: NewQueryClient redeclared in this block                                                                                                                                                                                                                               
	../../api/poktroll/gateway/query_grpc.pb 2.go:42:6: other declaration of NewQueryClient                                                                                                                                                                                                                           
../../api/poktroll/gateway/query_grpc.pb 3.go:76:6: QueryServer redeclared in this block                                                                                                                                                                                                                                  
	../../api/poktroll/gateway/query_grpc.pb 2.go:76:6: other declaration of QueryServer                                                                                                                                                                                                                              
../../api/poktroll/gateway/query_grpc.pb 3.go:86:6: UnimplementedQueryServer redeclared in this block                                                                                                                                                                                                                     
	../../api/poktroll/gateway/query_grpc.pb 2.go:86:6: other declaration of UnimplementedQueryServer                                                                                                                                                                                                                 
../../api/poktroll/gateway/query_grpc.pb 3.go:103:6: UnsafeQueryServer redeclared in this block                                                                                                                                                                                                                           
	../../api/poktroll/gateway/query_grpc.pb 2.go:103:6: other declaration of UnsafeQueryServer                                                                                                                                                                                                                       
../../api/poktroll/gateway/query_grpc.pb 3.go:107:6: RegisterQueryServer redeclared in this block                                                                                                                                                                                                                         
	../../api/poktroll/gateway/query_grpc.pb 2.go:107:6: other declaration of RegisterQueryServer                                                                                                                                                                                                                     
../../api/poktroll/gateway/query_grpc.pb 3.go:107:6: too many errors                                                                                                                                                                                                                                                      
# github.com/pokt-network/poktroll/api/poktroll/tokenomics                                                                                                                                                                                                                                                                
../../api/poktroll/tokenomics/query_grpc.pb 3.go:22:2: Query_Params_FullMethodName redeclared in this block                                                                                                                                                                                                               
	../../api/poktroll/tokenomics/query_grpc.pb 2.go:22:2: other declaration of Query_Params_FullMethodName                                                                                                                                                                                                           
../../api/poktroll/tokenomics/query_grpc.pb 3.go:28:6: QueryClient redeclared in this block                                                                                                                                                                                                                               
	../../api/poktroll/tokenomics/query_grpc.pb 2.go:28:6: other declaration of QueryClient                                                                                                                                                                                                                           
../../api/poktroll/tokenomics/query_grpc.pb 3.go:33:6: queryClient redeclared in this block                                                                                                                                                                                                                               
	../../api/poktroll/tokenomics/query_grpc.pb 2.go:33:6: other declaration of queryClient                                                                                                                                                                                                                           
../../api/poktroll/tokenomics/query_grpc.pb 3.go:37:6: NewQueryClient redeclared in this block                                                                                                                                                                                                                            
	../../api/poktroll/tokenomics/query_grpc.pb 2.go:37:6: other declaration of NewQueryClient                                                                                                                                                                                                                        
../../api/poktroll/tokenomics/query_grpc.pb 3.go:53:6: QueryServer redeclared in this block                                                                                                                                                                                                                               
	../../api/poktroll/tokenomics/query_grpc.pb 2.go:53:6: other declaration of QueryServer                                                                                                                                                                                                                           
../../api/poktroll/tokenomics/query_grpc.pb 3.go:60:6: UnimplementedQueryServer redeclared in this block                                                                                                                                                                                                                  
	../../api/poktroll/tokenomics/query_grpc.pb 2.go:60:6: other declaration of UnimplementedQueryServer                                                                                                                                                                                                              
../../api/poktroll/tokenomics/query_grpc.pb 3.go:71:6: UnsafeQueryServer redeclared in this block                                                                                                                                                                                                                         
	../../api/poktroll/tokenomics/query_grpc.pb 2.go:71:6: other declaration of UnsafeQueryServer                                                                                                                                                                                                                     
../../api/poktroll/tokenomics/query_grpc.pb 3.go:75:6: RegisterQueryServer redeclared in this block                                                                                                                                                                                                                       
	../../api/poktroll/tokenomics/query_grpc.pb 2.go:75:6: other declaration of RegisterQueryServer                                                                                                                                                                                                                   
../../api/poktroll/tokenomics/query_grpc.pb 3.go:79:6: _Query_Params_Handler redeclared in this block                                                                                                                                                                                                                     
	../../api/poktroll/tokenomics/query_grpc.pb 2.go:79:6: other declaration of _Query_Params_Handler                                                                                                                                                                                                                 
../../api/poktroll/tokenomics/query_grpc.pb 3.go:100:5: Query_ServiceDesc redeclared in this block                                                                                                                                                                                                                        
	../../api/poktroll/tokenomics/query_grpc.pb 2.go:100:5: other declaration of Query_ServiceDesc                                                                                                                                                                                                                    
../../api/poktroll/tokenomics/query_grpc.pb 3.go:100:5: too many errors                                                                                                                                                                                                                                                   
# github.com/pokt-network/poktroll/api/poktroll/supplier                                                                                                                                                                                                                                                                  
../../api/poktroll/supplier/query_grpc.pb 3.go:22:2: Query_Params_FullMethodName redeclared in this block                                                                                                                                                                                                                 
	../../api/poktroll/supplier/query_grpc.pb 2.go:22:2: other declaration of Query_Params_FullMethodName                                                                                                                                                                                                             
../../api/poktroll/supplier/query_grpc.pb 3.go:23:2: Query_Supplier_FullMethodName redeclared in this block                                                                                                                                                                                                               
	../../api/poktroll/supplier/query_grpc.pb 2.go:23:2: other declaration of Query_Supplier_FullMethodName                                                                                                                                                                                                           
../../api/poktroll/supplier/query_grpc.pb 3.go:24:2: Query_AllSuppliers_FullMethodName redeclared in this block                                                                                                                                                                                                           
	../../api/poktroll/supplier/query_grpc.pb 2.go:24:2: other declaration of Query_AllSuppliers_FullMethodName                                                                                                                                                                                                       
../../api/poktroll/supplier/query_grpc.pb 3.go:30:6: QueryClient redeclared in this block                                                                                                                                                                                                                                 
	../../api/poktroll/supplier/query_grpc.pb 2.go:30:6: other declaration of QueryClient                                                                                                                                                                                                                             
../../api/poktroll/supplier/query_grpc.pb 3.go:38:6: queryClient redeclared in this block                                                                                                                                                                                                                                 
	../../api/poktroll/supplier/query_grpc.pb 2.go:38:6: other declaration of queryClient                                                                                                                                                                                                                             
../../api/poktroll/supplier/query_grpc.pb 3.go:42:6: NewQueryClient redeclared in this block                                                                                                                                                                                                                              
	../../api/poktroll/supplier/query_grpc.pb 2.go:42:6: other declaration of NewQueryClient                                                                                                                                                                                                                          
../../api/poktroll/supplier/query_grpc.pb 3.go:76:6: QueryServer redeclared in this block                                                                                                                                                                                                                                 
	../../api/poktroll/supplier/query_grpc.pb 2.go:76:6: other declaration of QueryServer                                                                                                                                                                                                                             
../../api/poktroll/supplier/query_grpc.pb 3.go:86:6: UnimplementedQueryServer redeclared in this block                                                                                                                                                                                                                    
	../../api/poktroll/supplier/query_grpc.pb 2.go:86:6: other declaration of UnimplementedQueryServer                                                                                                                                                                                                                
../../api/poktroll/supplier/query_grpc.pb 3.go:103:6: UnsafeQueryServer redeclared in this block                                                                                                                                                                                                                          
	../../api/poktroll/supplier/query_grpc.pb 2.go:103:6: other declaration of UnsafeQueryServer                                                                                                                                                                                                                      
../../api/poktroll/supplier/query_grpc.pb 3.go:107:6: RegisterQueryServer redeclared in this block                                                                                                                                                                                                                        
	../../api/poktroll/supplier/query_grpc.pb 2.go:107:6: other declaration of RegisterQueryServer                                                                                                                                                                                                                    
../../api/poktroll/supplier/query_grpc.pb 3.go:107:6: too many errors                                                                                                                                                                                                                                                     
: exit status 1                                                                                                                                                                                                                                                                                                           
make: *** [localnet_regenesis] Error 1
```

Additionally, while attempting the [Clean Slate](https://dev.poktroll.com/infrastructure/localnet#clean-slate-nuclear-option) steps, I ran into these errors:
```
make go_develop_and_test
✔ Generated Go code
Processing module application
Modifying file: ./api/poktroll/application/genesis.pulsar.go
Modifying file: ./api/poktroll/application/query.pulsar.go
Processing module gateway
Processing module proof
Processing module service
Processing module session
Modifying file: ./api/poktroll/session/query.pulsar.go
Processing module shared
Processing module supplier
Processing module tokenomics
# github.com/pokt-network/poktroll/x/shared/types
../../../shared/types/service.pb 3.go:29:6: RPCType redeclared in this block
	../../../shared/types/service.pb 2.go:29:6: other declaration of RPCType
../../../shared/types/service.pb 3.go:32:2: RPCType_UNKNOWN_RPC redeclared in this block
	../../../shared/types/service.pb 2.go:32:2: other declaration of RPCType_UNKNOWN_RPC
../../../shared/types/service.pb 3.go:33:2: RPCType_GRPC redeclared in this block
	../../../shared/types/service.pb 2.go:33:2: other declaration of RPCType_GRPC
../../../shared/types/service.pb 3.go:34:2: RPCType_WEBSOCKET redeclared in this block
	../../../shared/types/service.pb 2.go:34:2: other declaration of RPCType_WEBSOCKET
../../../shared/types/service.pb 3.go:35:2: RPCType_JSON_RPC redeclared in this block
	../../../shared/types/service.pb 2.go:35:2: other declaration of RPCType_JSON_RPC
../../../shared/types/service.pb 3.go:36:2: RPCType_REST redeclared in this block
	../../../shared/types/service.pb 2.go:36:2: other declaration of RPCType_REST
../../../shared/types/service.pb 3.go:39:5: RPCType_name redeclared in this block
	../../../shared/types/service.pb 2.go:39:5: other declaration of RPCType_name
../../../shared/types/service.pb 3.go:47:5: RPCType_value redeclared in this block
	../../../shared/types/service.pb 2.go:47:5: other declaration of RPCType_value
../../../shared/types/service.pb 3.go:65:6: ConfigOptions redeclared in this block
	../../../shared/types/service.pb 2.go:65:6: other declaration of ConfigOptions
../../../shared/types/service.pb 3.go:68:2: ConfigOptions_UNKNOWN_CONFIG redeclared in this block
	../../../shared/types/service.pb 2.go:68:2: other declaration of ConfigOptions_UNKNOWN_CONFIG
../../../shared/types/service.pb 3.go:68:2: too many errors
# github.com/pokt-network/poktroll/x/gateway/types
../../../gateway/types/gateway.pb 3.go:27:6: Gateway redeclared in this block
	../../../gateway/types/gateway.pb 2.go:27:6: other declaration of Gateway
../../../gateway/types/gateway.pb 3.go:63:5: xxx_messageInfo_Gateway redeclared in this block
	../../../gateway/types/gateway.pb 2.go:63:5: other declaration of xxx_messageInfo_Gateway
../../../gateway/types/gateway.pb 3.go:85:5: fileDescriptor_3b77f5ee1eae982b redeclared in this block
	../../../gateway/types/gateway.pb 2.go:85:5: other declaration of fileDescriptor_3b77f5ee1eae982b
../../../gateway/types/gateway.pb 3.go:147:6: encodeVarintGateway redeclared in this block
	../../../gateway/types/gateway.pb 2.go:147:6: other declaration of encodeVarintGateway
../../../gateway/types/gateway.pb 3.go:175:6: sovGateway redeclared in this block
	../../../gateway/types/gateway.pb 2.go:175:6: other declaration of sovGateway
../../../gateway/types/gateway.pb 3.go:178:6: sozGateway redeclared in this block
	../../../gateway/types/gateway.pb 2.go:178:6: other declaration of sozGateway
../../../gateway/types/gateway.pb 3.go:299:6: skipGateway redeclared in this block
	../../../gateway/types/gateway.pb 2.go:299:6: other declaration of skipGateway
../../../gateway/types/gateway.pb 3.go:379:2: ErrInvalidLengthGateway redeclared in this block
	../../../gateway/types/gateway.pb 2.go:379:2: other declaration of ErrInvalidLengthGateway
../../../gateway/types/gateway.pb 3.go:380:2: ErrIntOverflowGateway redeclared in this block
	../../../gateway/types/gateway.pb 2.go:380:2: other declaration of ErrIntOverflowGateway
../../../gateway/types/gateway.pb 3.go:381:2: ErrUnexpectedEndOfGroupGateway redeclared in this block
	../../../gateway/types/gateway.pb 2.go:381:2: other declaration of ErrUnexpectedEndOfGroupGateway
../../../gateway/types/gateway.pb 3.go:381:2: too many errors
# github.com/pokt-network/poktroll/x/shared/types
../../../shared/types/service.pb 3.go:29:6: RPCType redeclared in this block
	../../../shared/types/service.pb 2.go:29:6: other declaration of RPCType
../../../shared/types/service.pb 3.go:32:2: RPCType_UNKNOWN_RPC redeclared in this block
	../../../shared/types/service.pb 2.go:32:2: other declaration of RPCType_UNKNOWN_RPC
../../../shared/types/service.pb 3.go:33:2: RPCType_GRPC redeclared in this block
	../../../shared/types/service.pb 2.go:33:2: other declaration of RPCType_GRPC
../../../shared/types/service.pb 3.go:34:2: RPCType_WEBSOCKET redeclared in this block
	../../../shared/types/service.pb 2.go:34:2: other declaration of RPCType_WEBSOCKET
../../../shared/types/service.pb 3.go:35:2: RPCType_JSON_RPC redeclared in this block
	../../../shared/types/service.pb 2.go:35:2: other declaration of RPCType_JSON_RPC
../../../shared/types/service.pb 3.go:36:2: RPCType_REST redeclared in this block
	../../../shared/types/service.pb 2.go:36:2: other declaration of RPCType_REST
../../../shared/types/service.pb 3.go:39:5: RPCType_name redeclared in this block
	../../../shared/types/service.pb 2.go:39:5: other declaration of RPCType_name
../../../shared/types/service.pb 3.go:47:5: RPCType_value redeclared in this block
	../../../shared/types/service.pb 2.go:47:5: other declaration of RPCType_value
../../../shared/types/service.pb 3.go:65:6: ConfigOptions redeclared in this block
	../../../shared/types/service.pb 2.go:65:6: other declaration of ConfigOptions
../../../shared/types/service.pb 3.go:68:2: ConfigOptions_UNKNOWN_CONFIG redeclared in this block
	../../../shared/types/service.pb 2.go:68:2: other declaration of ConfigOptions_UNKNOWN_CONFIG
../../../shared/types/service.pb 3.go:68:2: too many errors
# github.com/pokt-network/poktroll/x/gateway/types
../../../gateway/types/gateway.pb 3.go:27:6: Gateway redeclared in this block
	../../../gateway/types/gateway.pb 2.go:27:6: other declaration of Gateway
../../../gateway/types/gateway.pb 3.go:63:5: xxx_messageInfo_Gateway redeclared in this block
	../../../gateway/types/gateway.pb 2.go:63:5: other declaration of xxx_messageInfo_Gateway
../../../gateway/types/gateway.pb 3.go:85:5: fileDescriptor_3b77f5ee1eae982b redeclared in this block
	../../../gateway/types/gateway.pb 2.go:85:5: other declaration of fileDescriptor_3b77f5ee1eae982b
../../../gateway/types/gateway.pb 3.go:147:6: encodeVarintGateway redeclared in this block
	../../../gateway/types/gateway.pb 2.go:147:6: other declaration of encodeVarintGateway
../../../gateway/types/gateway.pb 3.go:175:6: sovGateway redeclared in this block
	../../../gateway/types/gateway.pb 2.go:175:6: other declaration of sovGateway
../../../gateway/types/gateway.pb 3.go:178:6: sozGateway redeclared in this block
	../../../gateway/types/gateway.pb 2.go:178:6: other declaration of sozGateway
../../../gateway/types/gateway.pb 3.go:299:6: skipGateway redeclared in this block
	../../../gateway/types/gateway.pb 2.go:299:6: other declaration of skipGateway
../../../gateway/types/gateway.pb 3.go:379:2: ErrInvalidLengthGateway redeclared in this block
	../../../gateway/types/gateway.pb 2.go:379:2: other declaration of ErrInvalidLengthGateway
../../../gateway/types/gateway.pb 3.go:380:2: ErrIntOverflowGateway redeclared in this block
	../../../gateway/types/gateway.pb 2.go:380:2: other declaration of ErrIntOverflowGateway
../../../gateway/types/gateway.pb 3.go:381:2: ErrUnexpectedEndOfGroupGateway redeclared in this block
	../../../gateway/types/gateway.pb 2.go:381:2: other declaration of ErrUnexpectedEndOfGroupGateway
../../../gateway/types/gateway.pb 3.go:381:2: too many errors
prog.go:12:2: no required module provides package github.com/golang/mock/mockgen/model: go.mod file not found in current directory or any parent directory; see 'go help modules'
prog.go:14:2: no required module provides package github.com/pokt-network/poktroll/x/application/types: go.mod file not found in current directory or any parent directory; see 'go help modules'
2024/03/25 09:40:08 Loading input failed: exit status 1
x/application/types/expected_keepers.go:1: running "mockgen": exit status 1
make: *** [go_mockgen] Error 1
```
## Conclusion
Please let me know if there is more info you'd like me to provide. I'll continue troubleshooting and report back with anything I find, and am happy to help test any fixes/ next steps. 
**Creator**: @0xThresh

**Comment by Olshansk** [comment]
@0xThresh I think we need to add a step where you have to run `make go_develop_and_test` before running `make localnet_up`. 

Can you try this out and let us know if it works?  If it does, can you update the docs?

**Comment by Olshansk** [comment]
@0xThresh Bump

**Comment by 0xThresh** [comment]
I tried running `make go_develop_and_test` a couple days ago and got a long list of errors that made it seem like it was an issue with my Go install. I did a fresh Go install and re-cloned, and this was what I ran into this time:

```
make go_develop_and_test
✔ Generated Go code
Processing module application
Modifying file: ./api/poktroll/application/genesis.pulsar.go
Modifying file: ./api/poktroll/application/query.pulsar.go
Processing module gateway
Processing module proof
Processing module service
Processing module session
Modifying file: ./api/poktroll/session/query.pulsar.go
Processing module shared
Processing module supplier
Processing module tokenomics
?   	github.com/pokt-network/poktroll/api/poktroll/application	[no test files]
?   	github.com/pokt-network/poktroll/api/poktroll/application/module	[no test files]
?   	github.com/pokt-network/poktroll/api/poktroll/gateway	[no test files]
?   	github.com/pokt-network/poktroll/api/poktroll/gateway/module	[no test files]
?   	github.com/pokt-network/poktroll/api/poktroll/proof	[no test files]
?   	github.com/pokt-network/poktroll/api/poktroll/proof/module	[no test files]
?   	github.com/pokt-network/poktroll/api/poktroll/service	[no test files]
?   	github.com/pokt-network/poktroll/api/poktroll/service/module	[no test files]
?   	github.com/pokt-network/poktroll/api/poktroll/session	[no test files]
?   	github.com/pokt-network/poktroll/api/poktroll/session/module	[no test files]
?   	github.com/pokt-network/poktroll/api/poktroll/shared	[no test files]
?   	github.com/pokt-network/poktroll/api/poktroll/supplier	[no test files]
?   	github.com/pokt-network/poktroll/api/poktroll/supplier/module	[no test files]
?   	github.com/pokt-network/poktroll/api/poktroll/tokenomics	[no test files]
?   	github.com/pokt-network/poktroll/api/poktroll/tokenomics/module	[no test files]
# github.com/pokt-network/poktroll/app.test
ld: warning: '/private/var/folders/xl/d_dcbj6s7w3gqb_jj2drh5fh0000gn/T/go-link-2090028950/000017.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
ok  	github.com/pokt-network/poktroll/app	2.201s
?   	github.com/pokt-network/poktroll/cmd/poktrolld	[no test files]
?   	github.com/pokt-network/poktroll/cmd/poktrolld/cmd	[no test files]
?   	github.com/pokt-network/poktroll/cmd/signals	[no test files]
?   	github.com/pokt-network/poktroll/docs	[no test files]
?   	github.com/pokt-network/poktroll/pkg/appgateserver	[no test files]
?   	github.com/pokt-network/poktroll/pkg/appgateserver/cmd	[no test files]
?   	github.com/pokt-network/poktroll/pkg/client	[no test files]
ok  	github.com/pokt-network/poktroll/pkg/appgateserver/config	1.272s
# github.com/pokt-network/poktroll/pkg/client/delegation.test
ld: warning: '/private/var/folders/xl/d_dcbj6s7w3gqb_jj2drh5fh0000gn/T/go-link-2312213908/000017.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
# github.com/pokt-network/poktroll/pkg/client/block.test
ld: warning: '/private/var/folders/xl/d_dcbj6s7w3gqb_jj2drh5fh0000gn/T/go-link-4177933501/000017.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
# github.com/pokt-network/poktroll/pkg/client/events.test
ld: warning: '/private/var/folders/xl/d_dcbj6s7w3gqb_jj2drh5fh0000gn/T/go-link-3851070092/000017.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
?   	github.com/pokt-network/poktroll/pkg/client/events/websocket	[no test files]
?   	github.com/pokt-network/poktroll/pkg/client/keyring	[no test files]
?   	github.com/pokt-network/poktroll/pkg/client/query	[no test files]
?   	github.com/pokt-network/poktroll/pkg/client/query/types	[no test files]
ok  	github.com/pokt-network/poktroll/pkg/client/block	3.015s
ok  	github.com/pokt-network/poktroll/pkg/client/delegation	2.155s
# github.com/pokt-network/poktroll/pkg/client/tx.test
ld: warning: '/private/var/folders/xl/d_dcbj6s7w3gqb_jj2drh5fh0000gn/T/go-link-2167397543/000017.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
# github.com/pokt-network/poktroll/pkg/client/supplier.test
ld: warning: '/private/var/folders/xl/d_dcbj6s7w3gqb_jj2drh5fh0000gn/T/go-link-3941427237/000017.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
?   	github.com/pokt-network/poktroll/pkg/client/tx/types	[no test files]
?   	github.com/pokt-network/poktroll/pkg/crypto	[no test files]
?   	github.com/pokt-network/poktroll/pkg/crypto/pubkey_client	[no test files]
ok  	github.com/pokt-network/poktroll/pkg/client/events	5.581s
ok  	github.com/pokt-network/poktroll/pkg/client/supplier	2.627s
# github.com/pokt-network/poktroll/pkg/crypto/rings.test
ld: warning: '/private/var/folders/xl/d_dcbj6s7w3gqb_jj2drh5fh0000gn/T/go-link-3442614493/000017.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
?   	github.com/pokt-network/poktroll/pkg/deps/config	[no test files]
?   	github.com/pokt-network/poktroll/pkg/observable	[no test files]
?   	github.com/pokt-network/poktroll/pkg/either	[no test files]
?   	github.com/pokt-network/poktroll/pkg/observable/filter	[no test files]
?   	github.com/pokt-network/poktroll/pkg/observable/logging	[no test files]
ok  	github.com/pokt-network/poktroll/pkg/client/tx	3.441s
?   	github.com/pokt-network/poktroll/pkg/partials/payloads	[no test files]
ok  	github.com/pokt-network/poktroll/pkg/crypto/rings	2.312s
ok  	github.com/pokt-network/poktroll/pkg/observable/channel	2.251s
ok  	github.com/pokt-network/poktroll/pkg/partials	1.662s
ok  	github.com/pokt-network/poktroll/pkg/polylog	1.863s
ok  	github.com/pokt-network/poktroll/pkg/polylog/polyzero	1.979s
# github.com/pokt-network/poktroll/pkg/relayer.test
ld: warning: '/private/var/folders/xl/d_dcbj6s7w3gqb_jj2drh5fh0000gn/T/go-link-1006231760/000017.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
?   	github.com/pokt-network/poktroll/pkg/relayer/cmd	[no test files]
# github.com/pokt-network/poktroll/pkg/relayer/protocol.test
ld: warning: '/private/var/folders/xl/d_dcbj6s7w3gqb_jj2drh5fh0000gn/T/go-link-2512520548/000017.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
# github.com/pokt-network/poktroll/pkg/relayer/miner.test
ld: warning: '/private/var/folders/xl/d_dcbj6s7w3gqb_jj2drh5fh0000gn/T/go-link-1776264777/000017.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
?   	github.com/pokt-network/poktroll/pkg/relayer/miner/gen	[no test files]
ok  	github.com/pokt-network/poktroll/pkg/relayer	1.539s
ok  	github.com/pokt-network/poktroll/pkg/relayer/config	1.727s
ok  	github.com/pokt-network/poktroll/pkg/relayer/miner	1.860s
ok  	github.com/pokt-network/poktroll/pkg/relayer/protocol	2.143s
# github.com/pokt-network/poktroll/pkg/relayer/proxy.test
ld: warning: '/private/var/folders/xl/d_dcbj6s7w3gqb_jj2drh5fh0000gn/T/go-link-3827076736/000017.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
# github.com/pokt-network/poktroll/pkg/relayer/session.test
ld: warning: '/private/var/folders/xl/d_dcbj6s7w3gqb_jj2drh5fh0000gn/T/go-link-643083442/000017.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
?   	github.com/pokt-network/poktroll/pkg/sdk	[no test files]
?   	github.com/pokt-network/poktroll/pkg/signer	[no test files]
?   	github.com/pokt-network/poktroll/testutil/application/mocks	[no test files]
?   	github.com/pokt-network/poktroll/testutil/gateway/mocks	[no test files]
?   	github.com/pokt-network/poktroll/testutil/keeper	[no test files]
?   	github.com/pokt-network/poktroll/testutil/mockclient	[no test files]
?   	github.com/pokt-network/poktroll/testutil/mockcrypto	[no test files]
?   	github.com/pokt-network/poktroll/testutil/mockrelayer	[no test files]
?   	github.com/pokt-network/poktroll/testutil/network	[no test files]
?   	github.com/pokt-network/poktroll/testutil/nullify	[no test files]
?   	github.com/pokt-network/poktroll/testutil/proof/mocks	[no test files]
?   	github.com/pokt-network/poktroll/testutil/sample	[no test files]
?   	github.com/pokt-network/poktroll/testutil/service/mocks	[no test files]
?   	github.com/pokt-network/poktroll/testutil/session/mocks	[no test files]
?   	github.com/pokt-network/poktroll/testutil/supplier/mocks	[no test files]
?   	github.com/pokt-network/poktroll/testutil/testchannel	[no test files]
?   	github.com/pokt-network/poktroll/testutil/testclient	[no test files]
?   	github.com/pokt-network/poktroll/testutil/testclient/testblock	[no test files]
?   	github.com/pokt-network/poktroll/testutil/testclient/testdelegation	[no test files]
?   	github.com/pokt-network/poktroll/testutil/testclient/testqueryclients	[no test files]
?   	github.com/pokt-network/poktroll/testutil/testclient/testsupplier	[no test files]
?   	github.com/pokt-network/poktroll/testutil/testclient/testtx	[no test files]
?   	github.com/pokt-network/poktroll/testutil/testcrypto/rings	[no test files]
?   	github.com/pokt-network/poktroll/testutil/testclient/testeventsquery	[no test files]
?   	github.com/pokt-network/poktroll/testutil/testerrors	[no test files]
?   	github.com/pokt-network/poktroll/testutil/testclient/testkeyring	[no test files]
?   	github.com/pokt-network/poktroll/testutil/testkeyring	[no test files]
?   	github.com/pokt-network/poktroll/testutil/testpolylog	[no test files]
?   	github.com/pokt-network/poktroll/testutil/testproxy	[no test files]
?   	github.com/pokt-network/poktroll/testutil/testrelayer	[no test files]
?   	github.com/pokt-network/poktroll/testutil/tokenomics/mocks	[no test files]
?   	github.com/pokt-network/poktroll/testutil/yaml	[no test files]
?   	github.com/pokt-network/poktroll/tools/scripts/goimports	[no test files]
?   	github.com/pokt-network/poktroll/tools/scripts/goimports/filters	[no test files]
# github.com/pokt-network/poktroll/x/application/types.test
ld: warning: '/private/var/folders/xl/d_dcbj6s7w3gqb_jj2drh5fh0000gn/T/go-link-4022279954/000017.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
# github.com/pokt-network/poktroll/x/application/module/config.test
ld: warning: '/private/var/folders/xl/d_dcbj6s7w3gqb_jj2drh5fh0000gn/T/go-link-2967296547/000017.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
# github.com/pokt-network/poktroll/x/application/keeper.test
ld: warning: '/private/var/folders/xl/d_dcbj6s7w3gqb_jj2drh5fh0000gn/T/go-link-2250780440/000017.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
# github.com/pokt-network/poktroll/x/gateway/module/config.test
ld: warning: '/private/var/folders/xl/d_dcbj6s7w3gqb_jj2drh5fh0000gn/T/go-link-3081729694/000017.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
# github.com/pokt-network/poktroll/x/gateway/keeper.test
ld: warning: '/private/var/folders/xl/d_dcbj6s7w3gqb_jj2drh5fh0000gn/T/go-link-3501481935/000017.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
# github.com/pokt-network/poktroll/x/application/module.test
ld: warning: '/private/var/folders/xl/d_dcbj6s7w3gqb_jj2drh5fh0000gn/T/go-link-226878622/000017.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
?   	github.com/pokt-network/poktroll/x/application/simulation	[no test files]
ok  	github.com/pokt-network/poktroll/pkg/relayer/proxy	5.438s
--- FAIL: TestRelayerSessionsManager_Start (0.39s)
    controller.go:137: missing call(s) to *mockclient.MockSupplierClient.CreateClaim(is equal to context.Background.WithValue(type string, val <not Stringer>).WithValue(type zerolog.ctxKey, val <not Stringer>) (*context.valueCtx), is assignable to SessionHeader, is assignable to ) /Users/jw/Documents/projects/poktroll/testutil/testclient/testsupplier/client.go:53
    controller.go:137: missing call(s) to *mockclient.MockSupplierClient.SubmitProof(is equal to context.Background.WithValue(type string, val <not Stringer>).WithValue(type zerolog.ctxKey, val <not Stringer>) (*context.valueCtx), is assignable to SessionHeader, is assignable to ) /Users/jw/Documents/projects/poktroll/testutil/testclient/testsupplier/client.go:62
    controller.go:137: aborting test due to missing call(s)
FAIL
FAIL	github.com/pokt-network/poktroll/pkg/relayer/session	2.354s
ok  	github.com/pokt-network/poktroll/pkg/retry	1.846s
# github.com/pokt-network/poktroll/x/gateway/module.test
ld: warning: '/private/var/folders/xl/d_dcbj6s7w3gqb_jj2drh5fh0000gn/T/go-link-1937595741/000017.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
?   	github.com/pokt-network/poktroll/x/gateway/simulation	[no test files]
ok  	github.com/pokt-network/poktroll/x/application/keeper	2.077s
# github.com/pokt-network/poktroll/x/gateway/types.test
ld: warning: '/private/var/folders/xl/d_dcbj6s7w3gqb_jj2drh5fh0000gn/T/go-link-643616539/000017.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
# github.com/pokt-network/poktroll/x/proof/types.test
ld: warning: '/private/var/folders/xl/d_dcbj6s7w3gqb_jj2drh5fh0000gn/T/go-link-3790769651/000017.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
# github.com/pokt-network/poktroll/x/proof/keeper.test
ld: warning: '/private/var/folders/xl/d_dcbj6s7w3gqb_jj2drh5fh0000gn/T/go-link-2021140927/000017.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
# github.com/pokt-network/poktroll/x/service/keeper.test
ld: warning: '/private/var/folders/xl/d_dcbj6s7w3gqb_jj2drh5fh0000gn/T/go-link-3378086007/000017.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
# github.com/pokt-network/poktroll/x/service/module.test
ld: warning: '/private/var/folders/xl/d_dcbj6s7w3gqb_jj2drh5fh0000gn/T/go-link-3406605976/000017.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
# github.com/pokt-network/poktroll/x/proof/module.test
ld: warning: '/private/var/folders/xl/d_dcbj6s7w3gqb_jj2drh5fh0000gn/T/go-link-3112459546/000017.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
?   	github.com/pokt-network/poktroll/x/proof/simulation	[no test files]
?   	github.com/pokt-network/poktroll/x/service/simulation	[no test files]
# github.com/pokt-network/poktroll/x/service/types.test
ld: warning: '/private/var/folders/xl/d_dcbj6s7w3gqb_jj2drh5fh0000gn/T/go-link-2405376815/000017.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
# github.com/pokt-network/poktroll/x/session/keeper.test
ld: warning: '/private/var/folders/xl/d_dcbj6s7w3gqb_jj2drh5fh0000gn/T/go-link-276217475/000017.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
# github.com/pokt-network/poktroll/x/session/types.test
ld: warning: '/private/var/folders/xl/d_dcbj6s7w3gqb_jj2drh5fh0000gn/T/go-link-1444521051/000017.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
# github.com/pokt-network/poktroll/x/session/module.test
ld: warning: '/private/var/folders/xl/d_dcbj6s7w3gqb_jj2drh5fh0000gn/T/go-link-2578412859/000017.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
?   	github.com/pokt-network/poktroll/x/session/simulation	[no test files]
?   	github.com/pokt-network/poktroll/x/shared/types	[no test files]
# github.com/pokt-network/poktroll/x/supplier/config.test
ld: warning: '/private/var/folders/xl/d_dcbj6s7w3gqb_jj2drh5fh0000gn/T/go-link-3738549658/000017.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
# github.com/pokt-network/poktroll/x/supplier/types.test
ld: warning: '/private/var/folders/xl/d_dcbj6s7w3gqb_jj2drh5fh0000gn/T/go-link-2523483050/000017.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
# github.com/pokt-network/poktroll/x/supplier/keeper.test
ld: warning: '/private/var/folders/xl/d_dcbj6s7w3gqb_jj2drh5fh0000gn/T/go-link-3523198832/000017.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
# github.com/pokt-network/poktroll/x/supplier/module.test
ld: warning: '/private/var/folders/xl/d_dcbj6s7w3gqb_jj2drh5fh0000gn/T/go-link-1363947950/000017.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
?   	github.com/pokt-network/poktroll/x/supplier/simulation	[no test files]
# github.com/pokt-network/poktroll/x/tokenomics/keeper.test
ld: warning: '/private/var/folders/xl/d_dcbj6s7w3gqb_jj2drh5fh0000gn/T/go-link-2476626918/000017.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
# github.com/pokt-network/poktroll/x/tokenomics/types.test
ld: warning: '/private/var/folders/xl/d_dcbj6s7w3gqb_jj2drh5fh0000gn/T/go-link-2418680036/000017.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
# github.com/pokt-network/poktroll/x/tokenomics/module.test
ld: warning: '/private/var/folders/xl/d_dcbj6s7w3gqb_jj2drh5fh0000gn/T/go-link-1667175456/000017.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
?   	github.com/pokt-network/poktroll/x/tokenomics/simulation	[no test files]
ok  	github.com/pokt-network/poktroll/x/application/module	96.633s
ok  	github.com/pokt-network/poktroll/x/application/module/config	1.389s
ok  	github.com/pokt-network/poktroll/x/application/types	2.584s
ok  	github.com/pokt-network/poktroll/x/gateway/keeper	3.589s
ok  	github.com/pokt-network/poktroll/x/gateway/module	51.081s
ok  	github.com/pokt-network/poktroll/x/gateway/module/config	2.647s
ok  	github.com/pokt-network/poktroll/x/gateway/types	2.867s
ok  	github.com/pokt-network/poktroll/x/proof/keeper	17.553s
ok  	github.com/pokt-network/poktroll/x/proof/module	97.876s
ok  	github.com/pokt-network/poktroll/x/proof/types	2.335s
ok  	github.com/pokt-network/poktroll/x/service/keeper	4.545s
ok  	github.com/pokt-network/poktroll/x/service/module	25.371s
ok  	github.com/pokt-network/poktroll/x/service/types	3.123s
ok  	github.com/pokt-network/poktroll/x/session/keeper	2.365s
ok  	github.com/pokt-network/poktroll/x/session/module	28.332s
ok  	github.com/pokt-network/poktroll/x/session/types	2.355s
ok  	github.com/pokt-network/poktroll/x/shared/helpers	2.874s
ok  	github.com/pokt-network/poktroll/x/supplier/client/cli	2.509s [no tests to run]
ok  	github.com/pokt-network/poktroll/x/supplier/config	1.728s
ok  	github.com/pokt-network/poktroll/x/supplier/keeper	2.090s
ok  	github.com/pokt-network/poktroll/x/supplier/module	71.912s
ok  	github.com/pokt-network/poktroll/x/supplier/types	1.881s
ok  	github.com/pokt-network/poktroll/x/tokenomics/keeper	2.104s
ok  	github.com/pokt-network/poktroll/x/tokenomics/module	8.390s
ok  	github.com/pokt-network/poktroll/x/tokenomics/types	1.467s
FAIL
make: *** [go_test] Error 1
```

Let me know if there's other info that would be helpful.

**Comment by 0xThresh** [comment]
I tried this again today on a different system using WSL2 instead of my M2 MacBook that I'd been trying on before. I was able to successfully start localnet just by using `make localnet_up`, and can confirm that I did not need to run `make_go_develop_and_test` first. 

Given that the issue seems specific to my Mac and the instructions in the documentation are accurate, I'm comfortable with this getting closed out. I can also provide more info/ try other steps on my MacBook if you think it's worth continuing to troubleshoot it.

**Comment by Olshansk** [comment]
@0xThresh Closing since you're unblocked and we can create a new ticket if need.
